src/libs/OptionsListUtils.js

```diff
@@ -133,7 +134,7 @@ function getPolicyExpenseReportOptions(report) {
  * @return {String}
  */
 function addSMSDomainIfPhoneNumber(login) {
-    if (Str.isValidPhone(login) && !Str.isValidEmail(login)) {
+    if (parsePhoneNumber(LoginUtils.appendCountryCode(login)).possible && !Str.isValidEmail(login)) {
         const smsLogin = login + CONST.SMS.DOMAIN;
         return smsLogin.includes('+') ? smsLogin : `+${countryCodeByIP}${smsLogin}`;
```

Comment:
That's not what I meant :sweat_smile:  The `appendCountryCode` should be called on the `smsLogin` but looking at this now, the function seems a little off, we should not add the country code since as you said login already includes the country code. That was done by https://github.com/Expensify/App/commit/ac357149f09bda0b395a6ee704ba9edbb3d762f8 where things were different.

Let's clear this up:
```js
function addSMSDomainIfPhoneNumber(login) {
    if (parsePhoneNumber(login).possible && !Str.isValidEmail(login)) {
        return login + CONST.SMS.DOMAIN;
    }
    return login;
}
```